src/main/scala/controllers/OpportunityController.scala

```diff
@@ -29,6 +30,17 @@ class OpportunityController @Inject()(opportunities: OpportunityOps, application
     }
   }
 
+  def createOpportunity () = Action { implicit request =>
+    request.getQueryString("type").getOrElse("") match {
+      case "new" =>
+        Ok(views.html.wip(routes.OpportunityController.createOpportunity().url))
+      case "reuse" =>
+        Ok(views.html.wip(routes.OpportunityController.createOpportunity().url))
+      case "" =>
+        Ok(views.html.showNewOpportunityForm())
```

Comment:
I'd rather avoid mixing "read-only" and actions, which actually modify some data.